client/ayon_resolve/plugins/publish/collect_plates.py

```diff
@@ -0,0 +1,32 @@
+import pyblish
+
+from ayon_resolve.otio import utils
+
+
+class CollectPlate(pyblish.api.InstancePlugin):
+    """Collect new plates."""
+
+    order = pyblish.api.CollectorOrder - 0.48
+    label = "Collect Plate"
+    hosts = ["resolve"]
+    families = ["plate"]
+
+    def process(self, instance):
+        """
+        Args:
+            instance (pyblish.Instance): The shot instance to update.
+        """
+        instance.data["folderPath"] = instance.data["folder_path"]
```

Comment:
Did we decide on keeping this `folder_path` instead of e.g. `targetFolderPath`?

Reply:
This code is no more relevant, hopefully this shed some light on this:
https://github.com/ynput/ayon-resolve/pull/15#discussion_r1746214679